### ISSUE TITLE
Suggested resource: postgresql_grant

### ISSUE THREAD

**Issue by danthegoodman** [terraform-providers/terraform-provider-postgresql#29]
I'd like to have support for managing [GRANT](https://www.postgresql.org/docs/10/static/sql-grant.html) and [REVOKE](https://www.postgresql.org/docs/10/static/sql-revoke.html) statements on individual roles. This could be similar to the [`mysql_grant`](https://www.terraform.io/docs/providers/mysql/r/grant.html) resource.

```hcl
resource "postgresql_grant" "bob-users" {
    role = "bob"
    tables = ["users"]
    privileges = ["SELECT"]
}

resource "postgresql_grant" "bob-sales" {
    role = "bob"
    tables = ["sales"]
    privileges = ["ALL"]
}
```

There's quite a few combinations of parameters which drive the `GRANT` command, so this resource could get pretty complicated.

**Comment by adharmawan** [comment]
The created resrource `postgresql_grant` only supports `ALL TABLES` or `ALL SEQUENCES` of a schema. Is there a plan to support specific table grants as per description above?

**Comment by cyrilgdn** [comment]
@adharmawan Yes, I can't really promise when it will be done but it was plan to be done in further iterations.

**Comment by galindro** [comment]
hi @cyrilgdn . Is there any plans to include this feature on the next release? I really need this feature...

**Comment by robertvanhoesel** [comment]
@jmks is implementing granular grants on tables in https://github.com/terraform-providers/terraform-provider-postgresql/pull/156 👍
